pkg/server/plugin/nodeattestor/x509pop/x509pop_test.go

```diff
@@ -157,7 +177,7 @@ func (s *Suite) TestAttestFailure() {
 		"x509pop: not configured")
 
 	// now configure
-	s.configure()
+	s.configure("")
```

Comment:
The empty string isn't my favourite thing in the world but it's not a blocker for me in test code.